core/asset/asset.go

```diff
@@ -118,7 +118,8 @@ func (reg *Registry) Define(ctx context.Context, xpubs []chainkd.XPub, quorum in
 		return nil, err
 	}
 
-	defhash := sha3.Sum256(rawDefinition)
+	var defhash bc.Hash
+	defhash.FromByte32(sha3.Sum256(rawDefinition))
```

Comment:
Having a NewHash function would make this code look more like what was here before:

```go
defhash := bc.NewHash(sha3.Sum256(rawDefinition))
```